pkgs/by-name/ju/just/setup-hook.sh

```diff
@@ -1,7 +1,10 @@
+# shellcheck shell=bash
```

Comment:
Love to see it (especially with no lints disabled)

Reply:
Out of curiosity, do we run shellcheck on CI for validation or this line is for manual editing?